tasks/libs/ciproviders/github_api.py

```diff
@@ -89,6 +89,39 @@ def get_milestone_by_name(self, milestone_name):
                 return milestone
         return None
 
+    def get_branch_protection_checks(self, branch_name: str) -> list[str]:
+        """
+        Get the required checks for a given branch
+        """
+        branch = self.get_branch(branch_name)
+        if not branch:
+            raise Exit(color_message(f"Branch {branch_name} not found", Color.RED), code=1)
+        elif not branch.protected:
+            raise Exit(color_message(f"Branch {branch_name} doesn't have required checks", Color.RED), code=1)
```

Comment:
nit
```suggestion
            raise Exit(color_message(f"Branch {branch_name} doesn't have protection", Color.RED), code=1)
```
I think protection is not equivalent to `required checks`. For instance release branches are protected but don't have any required checks
```
>>> b = d.get_branch('6.53.x')
>>> b.protected
True
>>> p = b.get_protection()
>>> p.required_status_checks
```

Reply:
Thanks, fixed with [35be02e](https://github.com/DataDog/datadog-agent/pull/31243/commits/35be02e3bfba8e775ed0836d67cb12ffd774f84d)!